desktop_app/start_application.py

```diff
@@ -0,0 +1,18 @@
+import sys
+from PyQt5.QtWidgets import QApplication
+
+sys.path.insert(0, 'GUI/Py')
+import ui # noqa
+
+
+def main():
+    app = QApplication(sys.argv)
+    loginWin = ui.LoginWindow()
+    loginWin.show()
+    app.exec_()
+    return 0
```

Comment:
LGTM. Only one small remark. I think it's better to return `app.exec_()` result instead of 0